doc/content/source/vectorpostprocessors/ResponseSpectraCalculator.md

```diff
@@ -9,13 +9,13 @@ For example, the input file below contains a `ResponseHistoryBuilder` VectorPost
 !listing test/tests/vectorpostprocessors/response_spectra_calculator/response_spectra_calculator.i block=VectorPostprocessors
 
 !alert note
-`ResponseSpectraCalculator` assumes that the response histories calculated in the vectorpostprocessor, provided in the `vectorpostprocessor` parameter in the input are acceleration histories, and will not perform any checks to ensure that they are accelerations. The user should ensure this by defining the corresponding `ResponseHistoryBuilder` appropriately. 
+`ResponseSpectraCalculator` assumes that the response histories calculated in the vectorpostprocessor, provided in the `vectorpostprocessor` parameter in the input are acceleration histories, and will not perform any checks to ensure that they are accelerations. The user should ensure this by defining the corresponding `ResponseHistoryBuilder` appropriately.
```

Comment:
I found the first sentence a bit harder to grasp, although I understood it. Please consider simplifying it.